docs/release_notes/v0.12.0-alpha.md

```diff
@@ -0,0 +1,73 @@
+# Spice v0.12.0-alpha (Apr 29, 2024)
+
+The 0.12.0-alpha release introduces Clickhouse and Spark data connectors and support for temporal refresh data periods. In addition, official Spice Client SDKs released with support for Spice OSS, and overall stability improvements have been added.
+
+## Highlights
+
+- **Clickhouse data connector**: Use Clickhouse databases as a data source.
+
+- **Spark data connector**: Use Apache Spark connections as a data source.
+
+- **Temporal data refresh period**: Set specific time windows for accelerated datasets refresh, ensuring only the most recent data retained.
+
+- **Optional ODBC data connector**: Use ODBC connections as a data source. This feature can be enabled by using the `odbc` cargo feature when building from the trunk.
+
+- **Spice Client SDK Support**: Official Spice.ai SDKs have been released with support for Spice OSS.
+  - [spice-rs v2.0.0](https://github.com/spiceai/spice-rs/releases/tag/v2.0.0)
+  - [spice.js v2.0.0](https://github.com/spiceai/spice.js/releases/tag/v2.0.0)
+  - [gospice v6.0.0](https://github.com/spiceai/gospice/releases/tag/v6.0.0)
+  - [spicepy v2.0.0](https://github.com/spiceai/spicepy/releases/tag/v2.0.0)
+
+## Contributors
+
+- @phillipleblanc
+- @Jeadie
+- @ewgenius
+- @sgrebnov
+- @y-f-u
+- @lukekim
+- @digadeesh
+- @gloomweaver
+- @edmondop
```

Comment:
Should add mach-kernel to this list too.

Reply:
Added